### PR TITLE
Release 0.11.3

### PR DESCRIPTION
**Pull request by dhatch**

### CODE REVIEW

**Review by Anniepoo — APPROVED** [review]
this all looks good to me - looks like ya changed the rev  numbers

**Review by plotnick — APPROVED** [review]
:ship:

File: languages/ruby/Gemfile.lock [hunk 1]
```diff
@@ -14,7 +14,7 @@ GEM
     coderay (1.1.3)
     diff-lcs (1.4.4)
     e2mmap (0.1.0)
-    ffi (1.14.2)
+    ffi (1.15.0)
```
Comment (gj) [inline languages/ruby/Gemfile.lock:17]: y

Comment (dhatch) [reply languages/ruby/Gemfile.lock:17]: just ran bundle install

Comment (dhatch) [reply languages/ruby/Gemfile.lock:17]: https://github.com/osohq/oso/blob/dfb2222bad002ea95ac18465f42460345b9774a4/languages/ruby/oso-oso.gemspec#L31

File: new-docs/content/any/project/changelogs/2021-03-31.md [hunk 2]
```diff
@@ -3,13 +3,13 @@ title: Release 2021-03-31
...
 ---
 
 ## `go-oso` 0.11.3
 
 ### Other bugs & improvements
 
-- Fixed a concurrency bug in the Go library causing panics when constructing new queries
\ No newline at end of file
+- Fixed a concurrency bug in the Go library causing panics when constructing new queries
+- The Go library now supports MUSL distros.
```
Comment (gj) [inline new-docs/content/any/project/changelogs/2021-03-31.md:15]: I think it's stylized as `musl` https://musl.libc.org/

```suggestion
- The Go library now supports musl distros.
```

File: languages/ruby/Gemfile.lock (hunk 1 above)
Comment (gj) [reply languages/ruby/Gemfile.lock:17]: ¯\\_(ツ)\_/¯

File: new-docs/content/any/project/changelogs/2021-03-31.md [hunk 3]
```diff
@@ -3,13 +3,13 @@ title: Release 2021-03-31
...
+  bug fixes & support for musl.
 ---
 
 ## `go-oso` 0.11.3
 
 ### Other bugs & improvements
 
-- Fixed a concurrency bug in the Go library causing panics when constructing new queries
\ No newline at end of file
+- Fixed a concurrency bug in the Go library causing panics when constructing new queries
```
Comment (gj) [inline new-docs/content/any/project/changelogs/2021-03-31.md:14]:
```suggestion
- Fixed a concurrency bug in the Go library causing panics when constructing new queries.
```

**Pull request merged**